rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -523,7 +523,8 @@ def build(self, *, doc_build_folder, output_staging_directory):
                 logger.info(
                     f"Running sphinx-apidoc: '{' '.join(cmd)}' in '{wrapped_sphinx_directory}'"
                 )
-                completed_process = subprocess.run(cmd, cwd=wrapped_sphinx_directory)
+                completed_process = subprocess.run(cmd, cwd=wrapped_sphinx_directory,
+                                                   stdout=sys.stdout, stderr=sys.stderr)
```

Comment:
Same behavior here looks the same as default.

Reply:
ditto